### PR TITLE
NOISSUE - Raise test coverage

### PR DESCRIPTION
**Pull request by dborovcanin**
Raise test coverage.

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
LGTM

**Review by anovakovic01 — CHANGES_REQUESTED** [review]

File: http/mocks/things.go [hunk 1]
```diff
@@ -6,10 +6,15 @@ import (
...
 	"github.com/mainflux/mainflux/things"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 var _ mainflux.ThingsServiceClient = (*thingsClient)(nil)
 
+// ErrToken is used to simulate internal server error.
+const ErrToken = "unavailable"
```
Comment (anovakovic01) [inline http/mocks/things.go]: Rename this to something else, this is convention for `error` type.

File: ws/mocks/things.go [hunk 2]
```diff
@@ -7,10 +7,15 @@ import (
...
 
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 var _ mainflux.ThingsServiceClient = (*thingsClient)(nil)
 
+// ErrToken is used to simulate internal server error.
+const ErrToken = "unavailable"
```
Comment (anovakovic01) [inline ws/mocks/things.go]: Same here.

**Review by anovakovic01 — CHANGES_REQUESTED** [review]
Almost there.

File: http/mocks/things.go (hunk 1 above)
Comment (dborovcanin) [reply http/mocks/things.go]: Irrelevant. If you find the more appropriate name, feel free to rename.

Comment (anovakovic01) [reply http/mocks/things.go]: It is not irrelevant, we should be consistent, even in mock. This name is confusing. Call it `UnavailableToken`, or something like that.

Comment (nmarcetic) [reply http/mocks/things.go]: I agree with @anovakovic01 I

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Pull request merged**